_sass/general.scss

```diff
@@ -86,15 +86,15 @@ a, .link {
     padding: 0 $pad;
 }
 
-// All post content is centred, except for text. On larger screens there's 25% spacing either side.
+// All post content is centred, except for text. On larger screens there's 40% spacing either side.
 .post-content {
     display: flex;
 
     flex-direction: column;
     align-items: center;
     @include respond-to(desktop) {
```

Comment:
I would use ```respond-to(large-screen)```, which is screens with over 1024px. It might be a bit too small on screens under 1000px